### PR TITLE
history&comment

### PR DESCRIPTION
**Pull request by mrtan1996**
#About

# Technical changes

# Relative URL

# UI changes

# TODO

# Others

### CODE REVIEW

**Comment by mrtan1996** [comment]
ready

File: resources/views/user/history.blade.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            <div class="history-table-cell history-table-col2">Mã đơn hàng</div>
+            <div class="history-table-cell history-table-col3">Thời gian</div>
+            <div class="history-table-cell history-table-col4">Địa điểm</div>
+            <div class="history-table-cell history-table-col5">Nhân viên</div>
+            <div class="history-table-cell history-table-col6">Tổng tiền</div>
+            <div class="history-table-cell history-table-col7">Trạng thái</div>
+            <div class="history-table-cell history-table-col8">Chi tiết</div>
+    </div>
+    </div>
+    </div> -->
```
Comment (Kenini1805) [inline resources/views/user/history.blade.php]: remove div commented

**Comment by Kenini1805** [comment]
commented

**Comment by mrtan1996** [comment]
ready

**Pull request merged**